examples/platform/nrfconnect/util/NFCWidget.cpp

```diff
@@ -52,26 +55,23 @@ int NFCWidget::StopTagEmulation()
 
     memset(mNdefBuffer, 0, sizeof(mNdefBuffer));
 
+    if (result)
```

Comment:
Wrong logic applied: https://github.com/nrfconnect/sdk-nrfxlib/blob/master/nfc/include/nfc_t2t_lib.h#L230

Maybe start using VerifyOrExit and check if result >= 0 as in `StartTagEmulation`.